docs/data/charts/styling/styling.md

```diff
@@ -164,3 +164,23 @@ Chart components accept the `sx` props.
 From here, you can target any subcomponents with its class name.
 
 {{"demo": "SxStyling.js"}}
+
+### Gradients and patterns
+
+It is possible to use gradients and patterns as fill for the charts.
+This can be done by using the `sx` prop in conjunction with passing your gradient or pattern definition as children of the chart component.
+
+{{"demo": "GradientBar.js"}}
```

Comment:
Do we need this example? Seems better to only advocate for inline gradient such that all elements can use it. especially after moving to HTML legend

Reply:
Hum, it is probably not needed anymore, since I changed the demo to use `vars` and the color prop, previously I was setting the css class, and the way to do that was very different from pie to the other charts.